resources/sass/pages/_plp.scss

```diff
@@ -0,0 +1,214 @@
+.product-listing-wrap .product-filter .product-categories {
```

Comment:
The CSS naming conventions and implementation has to be re-thought

Reply:
done